### ISSUE TITLE
Koski TL is not in Häme region

### ISSUE THREAD

**Issue by esakilpi** [juhariis/cds-fin-strawberry#1]
Koski TL (Turun Lääni) is not in Häme region but in Varsinais-Suomi.
The former municipality Koski HL has been renamed Hämeenkoski (61.0215° N, 25.1553° E)

**Comment by juhariis** [comment]
Right, I had to approximate as the crude logic I implemented handles only rectangles. Based on eyeballing the map and thinking about the geograpohic precision of the estimates I thought it was close enough.. :)
